enterprise/app/settings/user_github_link.tsx

```diff
@@ -122,8 +122,12 @@ export default class UserGitHubLink extends React.Component<Props, State> {
         <div className="settings-option-title">GitHub account link</div>
         <div className="settings-option-description">
           <p>
-            Linking a GitHub account allows you to configure BuildBuddy integrations for your GitHub repositories
-            directly in the BuildBuddy UI.
+            Linking your GitHub account to BuildBuddy lets you manage the BuildBuddy GitHub app directly in the
+            BuildBuddy UI.
+          </p>
+          <p>
+            Even if you login to BuildBuddy using your GitHub account, this must be explicitly granted to access
+            GitHub-related features.
```

Comment:
This feels slightly confusing since the settings page is called "GitHub account link."

I wonder if we should make this more explicit somehow - maybe having separate UI controls (or UI states) to indicate:
- GitHub account not linked at all
- GitHub account linked (only via gh login - so not fully authorized)
- GitHub account linked and authorized

Reply:
What do you think about adding a new section to the settings page called "GitHub settings" (in addition to Organization settings and Personal settings that we have now). And as sub-categories we could have 

"GitHub login"
"GitHub account link" 
"GitHub app installations"

I could do that in a followup

Reply:
I kind of prefer the current separation between org-level settings and user-level settings.

maybe we could add a separate "Account details" section under Personal settings, which would show your currently active login method. (This could be GitHub, gmail, SAML, etc., and it would always be read-only)

Reply:
Do you mind if I do that in a follow up?

Reply:
sgtm